woocommerce/woocommerce-gutenberg-products-block
# Fix radio control styles in editor

Fixes #1803.

### Screenshots

_Before:_
![imatge](https://user-images.githubusercontent.com/3616980/75325299-df116380-5878-11ea-8249-8fa8082e87e0.png)

_After:_
![imatge](https://user-images.githubusercontent.com/3616980/75325087-7d50f980-5878-11ea-8666-120279ecc656.png)

### How to test the changes in this Pull Request:

1. Add this code to `checkout/block.js` so it always renders radio inputs:
```JS
import RadioControl from '@woocommerce/base-components/radio-control';
...
			<RadioControl
				selected="1"
				options={ [
					{ value: '1', label: '1' },
					{ value: '2', label: '2' },
				] }
			/>
```
2. Create a post with a _Checkout_ block and verify the radio inputs look like in the designs.
